cura/Settings/GlobalStack.py

```diff
@@ -33,6 +35,7 @@ def __init__(self, container_id: str) -> None:
         super().__init__(container_id)
 
         self.setMetaDataEntry("type", "machine")  # For backward compatibility
+        self.setMetaDataEntry("group_id", str(uuid.uuid4()))  # Assign a new GlobalStack to a unique group by default
```

Comment:
We now still know what group means and what it does, but lets make sure that we still know it months / years from now. We should have some documentation here as to what the point of the group_id is and what it means.

Reply:
Done in 0cc00efa7983f18de6aa472be2f24b9c802a2630